File: examples/processors/get.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../.env.production")

reclient = Retab()

processor = reclient.processors.get(
    processor_id="proc_t51R0NeVxvWlBFEJJNVAd",
)

print(processor.model_dump_json(indent=2))

File: examples/typescript/processors/get.ts
import { config } from 'dotenv';
import { AsyncRetab } from '@retab/node';



async function main() {
  config();
  
  const reclient = new AsyncRetab();
  
  const processor = await reclient.processors.get("proc_Vq01vCzOnJctSkkrilzmJ");
  
  console.log(JSON.stringify(processor, null, 2));
  
}

main().catch(console.error);